Makefile

```diff
@@ -9,15 +9,16 @@ SHELL := bash
 BINDIR       := $(CURDIR)/bin
 BINNAME      ?= go-oscal
 INSTALL_PATH ?= /usr/local/bin
-OSCAL_LATEST := 1-1-1
-OSCAL_COMPONENT_SCHEMA_FILE := schema/component/oscal_component_schema-$(OSCAL_LATEST).json
-OSCAL_SSP_SCHEMA_FILE := schema/ssp/oscal_ssp_schema.json
+OSCAL_LATEST_SCHEMA := src/internal/schemas/oscal_complete_schema-$(OSCAL_LATEST).json
```

Comment:
Should we provide a sane default for OSCAL_LATEST ? Without looking at the docs below I was confused why the variable was removed yet still referenced.

Reply:
Yeah I was back and forth, the reason I removed was because it was another line of maintenance that is really only necessary when running the type generation, but I can see a good reason for having it there as an example.